portal-impl/src/com/liferay/portal/servlet/DirectServletRegistryImpl.java

```diff
@@ -223,8 +223,7 @@ protected void updateFileLastModified(String path, Servlet servlet) {
 	private static class PathModulePrefixHolder {
 
 		private static final String _PATH_MODULE_PREFIX = StringBundler.concat(
-			PortalUtil.getPathProxy(), PortalUtil.getPathContext(),
-			Portal.PATH_MODULE, StringPool.SLASH);
+			PortalUtil.getPathContext(), Portal.PATH_MODULE, StringPool.SLASH);
```

Comment:
@ChrisKian pathProxy was added to fix https://issues.liferay.com/browse/LPS-108204, why do you think it is safe to remove?

Reply:
@ChrisKian I see your point, it is safe to remove